zebra-network/src/peer_set/initialize/recent_by_ip.rs

```diff
@@ -0,0 +1,88 @@
+//! A set of IPs from recent connection attempts.
+
+use std::{
+    collections::{HashMap, VecDeque},
+    net::IpAddr,
+    time::{Duration, Instant},
+};
+
+use crate::constants;
+
+#[cfg(test)]
+mod tests;
+
+#[derive(Debug)]
+/// Stores IPs of recently attempted inbound connections.
+pub struct RecentByIp {
+    /// The list of IPs in increasing connection time order.
```

Comment:
New IPs are added to the back, and old IPs are split from the front.

I originally read "time" as "how long ago the connection was made" (connection age or connected time). But I think you meant "the instant the connection was made" (connection start or initiation time).

```suggestion
    /// The list of IPs in decreasing connection age order.
```

```suggestion
    /// The list of IPs in increasing connection initiation time order.
```

Reply:
I added this to PR #7213, we could also say both if you like, see PR #7214 as an example.

Reply:
> But I think you meant "the instant the connection was made"

Yep. I often struggle with these wordings, thank you for the assist.